front/app/api/projects_mini/types.ts

```diff
@@ -27,10 +30,10 @@ export interface MiniProjectData {
     action_descriptors: ActionDescriptors;
   };
   relationships: {
-    current_phase: {
+    current_phase?: {
       data: {
         id: string;
-      };
```

Comment:
For projects you follow, there's no guarantee that there is a current phase. So this can be `null`. For open to participation this was not the case.